www/docs/admin/alert_stats.php

```diff
@@ -9,6 +9,21 @@
 $PAGE->stripe_start();
 $PAGE->block_start(array ('id'=>'alerts', 'title'=>'Alert Statistics'));
 $db = new ParlDB;
+
+$q_confirmed = $db->query('select count(*) as c from alerts where confirmed and not deleted');
+$q_not_confirmed = $db->query('select count(*) as c from alerts where not confirmed and not deleted');
+$q_deleted = $db->query('select count(*) as c from alerts where confirmed and deleted');
+$q_speaker = $db->query('select count(*) as c from alerts where criteria like "speaker:%" and confirmed and not deleted');
+$q_keyword = $db->query("select count(*) as c from alerts where criteria not like '%speaker:%' and confirmed and not deleted");
+
+print '<h3>Headline stats</h3> <table>';
```

Comment:
There appears to be a `$PAGE->display_table` function that other admin pages use, might be worth using here?